java/src/jmri/jmrit/dispatcher/AutoTurnouts.java

```diff
@@ -343,6 +440,228 @@ private List<LayoutTrackExpectedState<LayoutTurnout>> turnoutUtil(Section s, int
         }
         return null;
     }
+//    /**
```

Comment:
There is a lot of new code in this file that's commented out. What's the purpose of that. Is it intended to be uncommented and used later?

Reply:
The blocks that I have commented out in AutoTurnouts have to be deleted but I have left them in for now for clarity. I will delete them.